gravitee-apim-console-webui/src/management/api/entrypoints-v4/edit/api-entrypoints-v4-edit.component.spec.ts

```diff
@@ -68,96 +72,316 @@ describe('ApiEntrypointsV4EditComponent', () => {
   const API = fakeApiV4({
     id: API_ID,
     listeners: [HTTP_LISTENER, SUBSCRIPTION_LISTENER],
+    endpointGroups: [
+      {
+        name: 'default-group',
+        type: 'kafka',
+        loadBalancer: {
+          type: 'ROUND_ROBIN',
+        },
+        endpoints: [
+          {
+            name: 'default',
+            type: 'kafka',
+            weight: 1,
+            inheritConfiguration: false,
+            configuration: {
+              bootstrapServers: 'localhost:9092',
+            },
+          },
+        ],
+      },
+      {
+        name: 'mock-group',
+        type: 'mock',
+        loadBalancer: {
+          type: 'ROUND_ROBIN',
+        },
+        endpoints: [EndpointV4Default.byType('mock')],
+      },
+    ],
   });
   const fakeUiRouter = { go: jest.fn() };
   let fixture: ComponentFixture<ApiEntrypointsV4EditComponent>;
   let httpTestingController: HttpTestingController;
   let loader: HarnessLoader;
 
-  const createComponent = (api: ApiV4) => {
+  const createComponent = (api: ApiV4, entrypointType: string) => {
     fixture = TestBed.createComponent(ApiEntrypointsV4EditComponent);
     fixture.detectChanges();
 
     expectGetEntrypoints();
     expectGetApi(api);
-    expectGetEntrypointSchema('http-get');
+    expectGetEntrypointSchema(entrypointType);
     fixture.detectChanges();
   };
 
-  beforeEach(() => {
-    TestBed.configureTestingModule({
-      imports: [NoopAnimationsModule, GioHttpTestingModule, ApiEntrypointsV4Module, MatIconTestingModule, MatAutocompleteModule],
-      providers: [
-        { provide: UIRouterStateParams, useValue: { apiId: API_ID, entrypointId: 'http-get' } },
-        { provide: UIRouterState, useValue: fakeUiRouter },
-      ],
+  describe('With HTTP GET entrypoint', () => {
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [NoopAnimationsModule, GioHttpTestingModule, ApiEntrypointsV4Module, MatIconTestingModule, MatAutocompleteModule],
+        providers: [
+          { provide: UIRouterStateParams, useValue: { apiId: API_ID, entrypointId: 'http-get' } },
+          { provide: UIRouterState, useValue: fakeUiRouter },
+        ],
+      });
+      httpTestingController = TestBed.inject(HttpTestingController);
+
+      createComponent(API, 'http-get');
+      fixture.detectChanges();
+      loader = TestbedHarnessEnvironment.loader(fixture);
     });
-    httpTestingController = TestBed.inject(HttpTestingController);
 
-    createComponent(API);
-    fixture.detectChanges();
-    loader = TestbedHarnessEnvironment.loader(fixture);
-  });
+    afterEach(() => {
+      httpTestingController.verify();
+    });
 
-  afterEach(() => {
-    httpTestingController.verify();
-  });
+    it('should show API configuration', async () => {
+      const formElement = fixture.debugElement.query(By.css('gio-form-json-schema'));
```

Comment:
Not related to PR, but it would be nice to have this in a harness for Particles as well as the configuration selector below, line 143.

Reply:
I'll create a ticket 😉